k8s/nginx-https/container/nginx.conf.template

```diff
@@ -158,8 +158,8 @@ stream {
   # Enable logging when connections are being throttled.
   limit_conn_log_level notice;
   
-  # Allow 16 connections from the same IP address.
-  limit_conn two 16;
+  # Allow 64 connections from the same IP address.
```

Comment:
Actually for a three node cluster, 64 is a sufficient number because connections via NGINX are under 64(connections between BDB, MDB across clusters). But I will bump it up to 256 or 512. So that we don't run out of limit fairly easily. @ttmc @krish7919 your take on this?